overlord/configstate/configstate.go

```diff
@@ -48,6 +48,25 @@ func ConfigureHookTimeout() time.Duration {
 
 // Configure returns a taskset to apply the given configuration patch.
 func Configure(st *state.State, snapName string, patch map[string]interface{}, flags int) *state.TaskSet {
+	summary := fmt.Sprintf(i18n.G("Run configure hook of %q snap"), snapName)
+
+	// configuration hook for "core" is handled internally
+	if snapName == "core" {
+		t := st.NewTask("configure-snapd", summary)
+		t.Set("patch", patch)
+		if flags&snapstate.UseConfigDefaults != 0 {
+			t.Set("use-defaults", true)
+		}
+		if flags&snapstate.IgnoreHookError != 0 {
+			t.Set("ignore-hook-error", true)
```

Comment:
there's no "hook" in this case, should we call the flags on the task just:  ignore-error, track-error ?